lib/engine/src/trap/frame_info.rs

```diff
@@ -127,14 +129,12 @@ impl GlobalFrameInfo {
             }
         };
 
-        // In debug mode for now assert that we found a mapping for `pc` within
-        // the function, because otherwise something is buggy along the way and
-        // not accounting for all the instructions. This isn't super critical
-        // though so we can omit this check in release mode.
-        debug_assert!(pos.is_some(), "failed to find instruction for {:x}", pc);
-
```

Comment:
It causes LLVM to fail because it doesn't emit the full trap information (that means, a map between a location and a trap type).

I explained a bit more a few lines after:

>             // Some compilers don't emit yet the full trap information for each of
>            // the instructions (such as LLVM).
>            // In case no specific instruction is found, we return by default the
>            // start offset of the function.